src/asynch/control.rs

```diff
@@ -182,7 +183,7 @@ impl<'a, const INGRESS_BUF_SIZE: usize, const URC_CAPACITY: usize>
         Ok(mac.to_be_bytes()[2..].try_into().unwrap())
     }
 
-    async fn get_wifi_status(&self) -> Result<WifiStatusVal, Error> {
```

Comment:
Any reason this should be public? The `WifiStatusVal` return type sounds like something internal?